generators/client-2/templates/src/main/webapp/app/components/alert/_alert.service.ts

```diff
@@ -1,14 +1,18 @@
 export function AlertService () {
     this.toast = false;
     /*jshint validthis: true */
-    this.$get = getService;
+    var service = {  getService: getService }
```

Comment:
Its a provider and hence that line is required. we need to see how we can migrate this properly